### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kai3341**

### CODE REVIEW

**Review by Jagrmi-C — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+[Way 3: The `taskcls` idea](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_03_taskcls_idea.py)
+--------
+
+You can launch worker by following command:
+
+```
+celery worker -P celery_pool_asyncio:TaskPool -A step_03_taskcls_idea
+```
+
+It looks like boilerplate problem has been solved
```
Comment (Jagrmi-C) [inline README.md]: a boilerplate

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+
+![Code Quality](https://import.viva64.com/docx/blog/0644_Haiku_3/image1.png)
+
+*Image source: [PVS-Studio corporate article](https://habr.com/en/company/pvs-studio/blog/461253/)*
+
+[Way 3: The `taskcls` idea](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_03_taskcls_idea.py)
+--------
+
+You can launch worker by following command:
+
```
Comment (Jagrmi-C) [inline README.md:48]: You can launch a worker by the following command:

File: README.md [hunk 3]
```diff
@@ -1,2 +1,57 @@
...
+[Way 2: Inheritance](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_02_inheritance.py)
+--------
+
+You can launch worker by following command:
+
+```
+celery worker -P celery_pool_asyncio:TaskPool -A step_02_inheritance
+```
+
+It's much better. The same behavior is implemented by the same code. But look to the [bottom of file](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_02_inheritance.py#L47). The Copy-Paste problem is not solved, and there are mistakes will be happened here.
```
Comment (Jagrmi-C) [inline README.md]: behaviour

File: README.md [hunk 4]
```diff
@@ -1,2 +1,57 @@
...
+
+![Code Quality](https://hsto.org/getpro/habr/post_images/df2/7f4/bcd/df27f4bcd139790b322570ee2f48e1ab.png)
+
+*Image source: [PVS-Studio corporate article](https://habr.com/en/company/pvs-studio/blog/472492/)*
+
+[Way 2: Inheritance](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_02_inheritance.py)
+--------
+
+You can launch worker by following command:
+
```
Comment (Jagrmi-C) [inline README.md:33]: You can launch a worker by the following command:

File: README.md [hunk 5]
```diff
@@ -1,2 +1,57 @@
...
+[Way 1: Simple Tasks](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_01_simple_tasks.py)
+--------
+
+You can launch worker by following command:
+
+```
+celery worker -P celery_pool_asyncio:TaskPool -A step_01_simple_tasks
+```
+
+There is no inheritance here. Just imagine -- you have to change behavior of these tasks. For example, you have to check response status code and retry the http request if response status is not equals 200. To reach it, you have to change the code of each task. Then imagine, the number of tasks is not 2, but 10, and you have to change each of them. Did you hear about [The Last Line Effect](https://www.viva64.com/en/b/0260/)? A mistake will definitely be made.
```
Comment (Jagrmi-C) [inline README.md]: is not equalled

File: README.md [hunk 6]
```diff
@@ -1,2 +1,57 @@
...
+--------
+
+There are three entrypoints containing the tasks with the same signatures and doing the same thing, but implemented in different way. Each of them contain two tasks. I want to explain why task inheritance is so important. Just imagine, you have to implement another 3 crawler tasks. You can compare ways and choose the most pretty to extend.
+
+
+[Way 1: Simple Tasks](https://github.com/kai3341/celery-decorator-taskcls-example/blob/master/step_01_simple_tasks.py)
+--------
+
+You can launch worker by following command:
+
```
Comment (Jagrmi-C) [inline README.md:18]: You can launch a worker by the following command:

File: README.md [hunk 7]
```diff
@@ -1,2 +1,57 @@
...
+===============
+
+This small project explains, how and why to use [celery-pool-asyncio](https://pypi.org/project/celery-pool-asyncio/) and [celery-decorator-taskcls](https://pypi.org/project/celery-decorator-taskcls/) libraries.
+
+* Free software: Apache Software License 2.0
+
+Structure
+--------
+
+There are three entrypoints containing the tasks with the same signatures and doing the same thing, but implemented in different way. Each of them contain two tasks. I want to explain why task inheritance is so important. Just imagine, you have to implement another 3 crawler tasks. You can compare ways and choose the most pretty to extend.
```
Comment (Jagrmi-C) [inline README.md]: in a different

**Review by Jagrmi-C — COMMENTED** [review]
Check

File: README.md (hunk 1 above)
Comment (kai3341) [reply README.md]: maybe, `the boilerplate`

**Review by Jagrmi-C — APPROVED** [review]

**Pull request merged**